### PR TITLE
python310Packages.ansible-runner: 2.3.1 -> 2.3.3

### PR DESCRIPTION
**Pull request by tjni**
###### Description of changes

https://github.com/ansible/ansible-runner/releases/tag/2.3.3
https://github.com/ansible/ansible-runner/releases/tag/2.3.2

Supersedes https://github.com/NixOS/nixpkgs/pull/219813.

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tjni** [comment]
@ofborg build python39Packages.ansible-runner python310Packages.ansible-runner python311Packages.ansible-runner

**Comment by tjni** [comment]
Result of `nixpkgs-review pr 240818` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.ansible-kernel</li>
    <li>python310Packages.ansible-kernel.dist</li>
    <li>python310Packages.ansible-runner</li>
    <li>python310Packages.ansible-runner.dist</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-kernel.dist</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.ansible-runner.dist</li>
  </ul>
</details>

**Comment by tjni** [comment]
Result of `nixpkgs-review pr 240818` run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.ansible-kernel</li>
    <li>python310Packages.ansible-kernel.dist</li>
    <li>python310Packages.ansible-runner</li>
    <li>python310Packages.ansible-runner.dist</li>
    <li>python311Packages.ansible-kernel</li>
    <li>python311Packages.ansible-kernel.dist</li>
    <li>python311Packages.ansible-runner</li>
    <li>python311Packages.ansible-runner.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/ansible/core.nix [hunk 1]
```diff
@@ -64,7 +69,16 @@ buildPythonPackage rec {
...
+  ] ++ lib.optionals windowsSupport [
+    pywinrm
+  ] ++ lib.optionals (pythonOlder "3.10") [
+    importlib-resources
+  ];
+
+  pythonRelaxDeps = if (pythonOlder "3.10") then
+    ["importlib-resources"]
+  else
+    null;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/ansible/core.nix:81]:
```suggestion
  pythonRelaxDeps = lib.optionals (pythonOlder "3.10") [
    "importlib-resources"
  ];
```

Comment (tjni) [reply pkgs/development/python-modules/ansible/core.nix:81]: Done.

**Pull request merged**